src/quo/animated.cljs

```diff
@@ -103,13 +103,13 @@
             else-node))))
 
 (defn block [opts]
-  (.block ^js animated (clj->js opts)))
+  (.block ^js animated (to-array opts)))
```

Comment:
could you elaborate please?

Reply:
the block will always receive an array of animated nodes, all nodes are already JS object. So this is just a micro-optimization because to-array iterate faster then clj->js.